react/ui-components/src/atoms/MultiSelectDropdown.js

```
@@ -54,7 +54,7 @@ const MultiSelectDropdown = ({
         //   );
         //   return updatedState;
       case "REMOVE_FROM_SELECTED_EVENT_QUEUE":
-        const newState = state.filter(
+        const newState = state?.filter(
           (e) => e?.code !== action.payload?.[1]?.code
         );
```

Biome

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Unsafe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)